edxval/tests/test_api.py

```diff
@@ -901,7 +908,9 @@ def test_existing_video_in_destination_course_id(self):
 
 @ddt
 class ExportTest(TestCase):
```

Comment:
We are missing a test case that asserts the exported transcript XML along with the content in the file system.

Reply:
Added